README.md

```diff
@@ -46,18 +46,30 @@ Overall progress is tracked via our [Project Tracking Board](https://github.com/
 
 ### Monitoring Websites
 
-| Tool Name | Description | Status |
-|-----------|-------------|--------|
-| [**Page Freezer CLI**](https://github.com/edgi-govdata-archiving/pagefreezer-cli) | CLI tools & data management protocols for interacting with Page Freezer | **In Progress** |
-
-#### Filtering Changes
+These repositories support the current workflow, based on Google spreadsheets
+automatically generated from scraping the Verionista web interface. They will be
+deprecated when a web app-based workflow is ready to use.
 
 | Tool Name | Description | Status |
 |-----------|-------------|--------|
 | [**Version Tracking UI**](https://github.com/edgi-govdata-archiving/version-tracking-ui) | Tools to facilitate the tracking website changes | **In Progress** |
```

Comment:
Is this project abandoned? Last commit was 21 days ago, and it's not clear to me whether it's supporting the old workflow (as listed here) or targeting the new one.